pnpm-workspace.yaml

```diff
@@ -8,3 +8,18 @@ onlyBuiltDependencies:
   - msw
   - node-pty
   - protobufjs
+# Without this, electron-builder will pull in electron-builder-squirrel-windows (which we don't need)
+# and that dep will in turn pull electron-winstaller (which we also don't need). pnpm will then
+# start complaining that electron-winstaller is not in onlyBuiltDependencies.
+#
+# We could instead solve this by adding electron-winstaller to ignoredBuiltDependencies, but why
+# pull in deps that we don't use? Ultimately, the problem is that electron-builder-squirrel-windows
+# is defined as a peer dep of one of electron-builder deps, while in reality it's more like an
+# optional dep.
+autoInstallPeers: false
```

Comment:
eslint-plugin-import is now missing, I'll have to look up what version we currently have on v17. It's weird though that `pnpm i --resolution-only` doesn't show any warning/error about this, I have to run `pnpm eslint` to see it.

Reply:
It's because `eslint-plugin-import` is an optional dependency https://github.com/import-js/eslint-import-resolver-typescript/blob/e36a0acebfd2846b58cc55c93db17e4d17b583cb/package.json#L73 💀 

I don't understand why, `eslint-import-resolver-typescript` needs `eslint-plugin-import` to work.